tests/phpunit/tests/theme/wpThemeGetBlockPatterns.php

```diff
@@ -12,6 +12,24 @@
  * @covers WP_Theme::get_block_patterns
  */
 class Tests_Theme_WPThemeGetBlockPatterns extends WP_UnitTestCase {
+	/**
+	 * The initial cache object.
+	 *
+	 * @var object
+	 */
+	private $initial_cache_object;
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->initial_cache_object = wp_using_ext_object_cache();
+	}
+
+	public function tear_down() {
+		parent::tear_down();
+
+		wp_using_ext_object_cache( $this->initial_cache_object );
```

Comment:
For tear down the order ought to be [specific then general](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#shared-fixtures) 

```suggestion
		wp_using_ext_object_cache( $this->initial_cache_object );
		parent::tear_down();
```